<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dolorsChatRoomServ</string>
  </property>
  <widget class="QTableWidget" name="onlineUsrDB">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>hostName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IpAddress</string>
    </property>
   </column>
  </widget>
  <widget class="QTextBrowser" name="statusBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>51</y>
     <width>256</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="userNumLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>在线人数：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
